app/src/main/java/org/literacyapp/authentication/StudentImageCollectionActivity.java

```diff
@@ -61,6 +62,7 @@
     private static final boolean DIAGNOSE_MODE = true;
     private static final long TIMER_DIFF = 200;
     private static final int NUMBER_OF_IMAGES = 20;
+    private static final int MAX_TIME_BEFORE_FALLBACK = 5000;
```

Comment:
Not exactly. The timer is reset if a face is detected anywhere on the screen. So the 5 seconds are if no face is detected at all on the whole screen. But also here it depends on how good the face detection will work in the end, so we can adjust the number.